lib/griddler/email.rb

```diff
@@ -86,6 +92,10 @@ def extract_cc_from_headers(headers)
       EmailParser.extract_cc(headers)
     end
 
+    def extract_spam_score
+      @spam_report['score'] || @spam_report[:score] if spam_report
```

Comment:
I think the adapters should be normalizing this attribute. We don't need to support a string and a symbol key. The symbol will work fine.